packages/cardano-services/test/load/load.test.ts

```diff
@@ -244,7 +245,16 @@ describe('load', () => {
           const coins = 1_000_000n + 1000n * BigInt(i);
           const tx = await wallet.initializeTx({ outputs: new Set([{ address, value: { coins } }]) });
 
-          submitPromises.push(wallet.submitTx(await wallet.finalizeTx(tx)));
+          submitPromises.push(
+            // eslint-disable-next-line no-loop-func
+            (async () => {
+              try {
```

Comment:
Is this to be able to see the actual error? If so, it's a [jest bug](https://github.com/facebook/jest/issues/11617) that will hopefully be fixed one day, and we shouldn't be adapting our tests to it. There's a workaround: `node scripts/patch-jest-worker.js`

Reply:
I'm sorry but I have not clear about which piece of code you are referring. Another thing I've not clear is that the workaround you are proposing seems to be about `child_process` related stuff while the `Promise`s in `submitPromises` are about asynchronous tasks not working with `child_process`.

Anyway I added that `try/catch block` when I faced the problem related to value is conserved error we are facing on another thread.
It has two benefits (when a transaction submission unexpectedly fails):

1. the tests do not hangs
2. we can have the evidence of reason of the error

Reply:
try/catch just seems odd there. How about: `await expect(promise).resolves.not.toThrow()`?

We can rely on test runner reports for failed expectations